backend/src/main/kotlin/cz/loono/backend/schedule/PlanNewExamReminderTask.kt

```diff
@@ -0,0 +1,45 @@
+package cz.loono.backend.schedule
+
+import cz.loono.backend.api.service.PreventionService
+import cz.loono.backend.api.service.PushNotificationService
+import cz.loono.backend.db.repository.AccountRepository
+import org.springframework.stereotype.Component
+import java.time.LocalDate
+import java.time.Period
+
+@Component
+class PlanNewExamReminderTask(
+    private val accountRepository: AccountRepository,
+    private val preventionService: PreventionService,
+    private val notificationService: PushNotificationService
+) : DailySchedulerTask {
+
+    override fun run() {
+        val today = LocalDate.now()
+        val accounts = accountRepository.findAll()
```

Comment:
I'm afraid we need to paginate over accounts, otherwise we might run into a problem when we have a lot of accounts and our app would fail on OOM, there is a function in `BadgeDowngradeTask`#`paginateOverAccounts`, we can extract it to `AccountsService` and reuse in both places